great_expectations/render/renderer/v3/suite_profile_notebook_renderer.py

```diff
@@ -192,13 +192,8 @@ def _add_user_configurable_profiler_cells(self) -> None:
     # },
     # column_value_nonnullity_rule={
     # },
-    # numeric_columns_low_precision_rule={
-    #     "round_decimals": 1,   # parameters will be rounded to 1 decimal place for metrics that preserve precision
-    #     "false_positive_rate": 0.1,
-    #     "random_seed": 43792,
-    # },
-    # numeric_columns_high_precision_rule={
-    #     "round_decimals": 4,   # parameters will be rounded to 4 decimal places for metrics that increase precision
+    # numeric_columns_rule={
+    #     "round_decimals": 4,
```

Comment:
This should be 12 now right?

Reply:
@NathanFarmer The 4 worked, because the primary culprit was the fact that `Expectation` implementation was deficient.  But I like the idea of 12 -- changed...  Pleas re-review.  Thanks!